react/daily-art-prompt/src/components/ImageLayout.test.js

```diff
@@ -64,16 +68,51 @@ describe('<ImageLayout />', () => {
     })
 
     it('should pass correct comment prop', () => {
-      const wrapper = shallow(<ImageLayout {...defaultProps} />)
       expect(wrapper.find({ 'data-testid': 1 }).prop('comment')).toEqual(defaultProps.image.comments[0])
     })
-    describe('when clicked', () => {
-      it('should call deleteComment with correct params', () => {
-        const wrapper = shallow(<ImageLayout {...defaultProps} />)
-        wrapper.find({ 'data-testid': 1 }).simulate('click')
+
+    describe('onDelete', () => {
+      it('should call deleteComment with correct params onDelete', () => {
+        wrapper.find({ 'data-testid': 1 }).simulate('delete')
         expect(defaultProps.deleteComment).toHaveBeenCalledWith(1, 1)
```

Comment:
it would make more sense to me, if you were checking:

```js
expect(defaultProps.deleteComment).toHaveBeenCalledWith(defaultProps.image.id, 1)
```